info.yml

```diff
@@ -76,6 +76,14 @@ configuration:
                      setting is an empty string, no logo will be applied.
         default_value: ""
 
+    hook_reviewsubmission_quicktime_settings:
```

Comment:
The name feels too long. Why not just call it `hook_quicktime_settings`? The reviewsubmission is implied by the app.